lib/googleauth/credentials.rb

```diff
@@ -26,6 +26,14 @@ module Auth
     # In most cases, it is subclassed by API-specific credential classes that
     # can be instantiated by clients.
     #
+    # **Important:** If you accept a credential configuration (credential
+    # JSON/File/Stream) from an external source for authentication to Google
+    # Cloud Platform, you must validate it before providing it to any Google
+    # API or library. Providing an unvalidated credential configuration to
+    # Google APIs can compromise the security of your systems and data.
+    # For more information, refer to [Validate credential configurations from
+    # external sources](https://cloud.google.com/docs/authentication/external/externally-sourced-credentials).
+    #
```

Comment:
can you also update https://github.com/googleapis/google-auth-library-ruby/blob/043ecb7c5e2f3a6d047421a2d3c3f8a79a1c8653/README.md

Reply:
done